jcl/src/openj9.jvm/share/classes/com/ibm/jvm/InvalidDumpOptionException.java

```diff
@@ -1,8 +1,6 @@
 /*[INCLUDE-IF Sidecar17]*/
-package com.ibm.jvm;
```

Comment:
While you're here, please change `Sidecar17` to `Sidecar18-SE`.

Reply:
Gladly.